### ISSUE TITLE
Unable to call EmailComposer.init()

### ISSUE THREAD

**Issue by dutchman71** [FrazzApps/Xamarin.Plugins#1]
Hi,

I must be doing something wrong. I installed EmailComposer in VS using NuGet Package Manager. In my class file I import the following:
using FrazzApps.Xamarin.EmailComposer;
using FrazzApps.Xamarin.EmailComposer.Abstractions;

Then I'm trying to initialize the composer as shown below. However, I get an error 'EmailComposer' doesn't exist in this context.

Any ideas?

Thanks!
        private void OnEmailClicked()
        {
            Debug.WriteLine("Email clicked");

```
        try
        {

            // Get email address
            var emailAddress = employee.Email;

            // Open Email composer
            EmailComposer.Init();
            IEmailComposer emailComposer = DependencyService.Get<IEmailComposer>();
            emailComposer.SendEmail(emailAddress, Globals.CONTACT_EMAIL_SUBJECT, "");


        }
        catch (Exception ex)
        {
            Debug.WriteLine("Exception " + ex.Message);
        }
        finally
        {

        }
    }
```

**Comment by FrazzApps** [comment]
Yes, that one got me too.  You need to call EmailComposer.Init();  in each of the device startups.  So in the Android mainActivity.cs. Or in the Windows Phone MainPage.xaml.cs.
If that isn't in the readme I will add it tonight.
Thanks for downloading and good luck with you app!

Cheers
Fraser

Ps any feedback\suggestions on tho or any of my other plugins is greatly appreciated

Sent from my Windows Phone

**Comment by dutchman71** [comment]
thanks very much for the info. Let me try that.

Chris
